quarkus/defaults/src/main/resources/application.properties

```diff
@@ -65,7 +65,7 @@ quarkus.micrometer.enabled=true
 quarkus.micrometer.export.prometheus.enabled=true
 
 quarkus.otel.enabled=true
```

Comment:
Should we turn `quarkus.otel.enabled` off as well? So that it disables all OpenTelemetry functionalities?

Reply:
I would advise against it: `quarkus.otel.enabled` is a build-time property. If you set it to false, all artifacts and Docker images will have telemetry disabled _forever_: it wouldn't be possible to turn it back on at run time.

Reply:
I'm not familiar with Quarkus, but I doubt this is a build-time property. Build-time properties are set during the build and can’t be changed at runtime. If it’s configurable in application.properties, it’s likely a runtime property instead.

Looked a bit more, it’s an actually build-time property per https://quarkus.io/guides/opentelemetry. This is confusing for users and developers. I think what we can do is to separate out or comment out these build-time properties. I don’t want users confused by setting them at the runtime and expecting they are working that way. We will need more discussion on how we provide a consistent way for Polaris users to configure the server. Let me bring it up to dev list later.
cc @eric-maynard @collado-mike @dennishuo @MonkeyCanCode